libs/zzz/page-optimize/src/CharSheetsDisplay.tsx

```diff
@@ -59,7 +59,7 @@ const s0AnbyDocs: Document[] = [
   {
```

Comment:
can you update the text above to indicate it reads from her core skill level now